addon/components/help-widget/help.js

```diff
@@ -13,7 +13,8 @@ export default class HelpWidgetHelpComponent extends Component {
   @tracked suggestions = [];
 
   get helpCenterLink() {
-    const url = this.intl.t('ember-help-widget.help-center-url');
+    const helpUrlkey = this.args?.helpCenterUrlKey || 'help-center-url';
+    const url = this.intl.t('ember-help-widget.'.concat(helpUrlkey));
```

Comment:
⬇️ ⬇️ You can actually use ES6 backticks string interpolation here, and it's a bit more readable:
```
this.intl.t(`ember-help-widget.${helpUrlkey}`);
```